libraries.cmake/sqlite.cmake

```diff
@@ -0,0 +1,99 @@
+##################################################
+###       SQLITE   															 ###
+##################################################
+
+MACRO( OPENMS_CONTRIB_BUILD_SQLITE )
+  OPENMS_LOGHEADER_LIBRARY("SQLITE")
+  #extract: (takes very long.. so skip if possible)
+  if(MSVC)
+    set(ZIP_ARGS "x -y -osrc")
+  else()
+    set(ZIP_ARGS "xzf")
+  endif(MSVC)
+  OPENMS_SMARTEXTRACT(ZIP_ARGS ARCHIVE_SQLITE "SQLITE" "INSTALL")
+  
+  if(MSVC)
+    set(MSBUILD_ARGS_TARGET "sqlite")
+    OPENMS_BUILDLIB("SQLITE (Debug)" MSBUILD_ARGS_SLN MSBUILD_ARGS_TARGET "Debug" SQLITE_DIR)
```

Comment:
yes, this very clearly will not work. there is no solution file set, e.g. `MSBUILD_ARGS_SLN` is empty!